docs/content/components/markdown.md

```diff
@@ -457,7 +457,7 @@ Inline code inside table cells should still be distinguishable.
 
 | Language    | Code               |
 |-------------|--------------------|
-| Javascript  | `var foo = "bar";` |
+| JavaScript  | `var foo = "bar";` |
```

Comment:
JavaScript is a brand, but sometimes certain things insist on writing it incorrectly. I'm hoping this instance doesn't care.